remote/remote_state_s3.go

```diff
@@ -42,6 +42,7 @@ type ExtendedRemoteStateConfigS3 struct {
 	EnableLockTableSSEncryption bool              `mapstructure:"enable_lock_table_ssencryption"`
 	DisableAWSClientChecksums   bool              `mapstructure:"disable_aws_client_checksums"`
 	AccessLoggingBucketName     string            `mapstructure:"accesslogging_bucket_name"`
+	AccessLoggingTargetPrefix   string            `mapstructure:"accesslogging_target_prefix"`
```

Comment:
I believe the original default was `TFStateLogs/`. Is the default now empty string?

Reply:
Yep, you're right that it had a default value - I've now decided to leave it explicitly to the user's control - so it is a bt more intuitive - if there's no value set, then they don't get a prefix set - and I've considered this (so far) as a potential decision for the user.

_I'd be grateful to see your take on this 🙂  Do you think it's better if we have the default value?_

Reply:
It's a trade-off, so it's a judgment call. Keeping the prefix would be backwards compatible, so all future buckets are created the way all previous ones were, which I would guess is a bit more preferable to not having a default at all in the future...